index.html

```diff
@@ -30,7 +30,6 @@ <h1>Hello world ,we are software devlopers!</h1>
           <li></li>
           <li></li>
           <li></li>
-          <li></li>
```

Comment:
Kindly use Html tags that are necessary for your project. empty bulletins didn't make sense for your audience.